src/index.js

```diff
@@ -32,6 +33,16 @@ registerPlugin( 'wc-calypso-bridge', {
 	),
 } );
 
+
+if ( !! window.wcCalypsoBridge.isEcommercePlanTrial ) {
+	registerPlugin( 'my-tasklist-footer-extension', {
+		render: DisabledTasksFill,
+		// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+		// @ts-ignore
```

Comment:
We can remove lines 40-41 since this is a JS file. 🙂

Reply:
Updated in https://github.com/Automattic/wc-calypso-bridge/pull/940/commits/ba23fabf93f3f7fee88fe3291c23cc6399b99e0a